pywr/parameters/_parameters.pyx

```diff
@@ -802,6 +847,8 @@ cdef class AnnualHarmonicSeriesParameter(Parameter):
         self._phase_lower_bounds = np.ones(n)*kwargs.pop('phase_lower_bounds', 0.0)
         self._phase_upper_bounds = np.ones(n)*kwargs.pop('phase_upper_bounds', np.pi*2)
         super(AnnualHarmonicSeriesParameter, self).__init__(model, *args, **kwargs)
+        # Size must be set after call to super.
+        self.size = 1 + 2*n
```

Comment:
Should this be using self.double_size?

Reply:
I've fixed this. There was also another reference in `ConstantParameter`.